src/main/java/formflow/library/ScreenController.java

```diff
@@ -49,609 +51,611 @@
 @RequestMapping("/flow")
 public class ScreenController extends FormFlowController {
 
-  private final List<FlowConfiguration> flowConfigurations;
-  private final ValidationService validationService;
-
-  private final AddressValidationService addressValidationService;
-
-  private final ConditionManager conditionManager;
-  private final ActionManager actionManager;
-
-  public ScreenController(
-      List<FlowConfiguration> flowConfigurations,
-      SubmissionRepositoryService submissionRepositoryService,
-      ValidationService validationService,
-      AddressValidationService addressValidationService,
-      ConditionManager conditionManager,
-      ActionManager actionManager) {
-
-    super(submissionRepositoryService);
-    this.flowConfigurations = flowConfigurations;
-    this.validationService = validationService;
-    this.addressValidationService = addressValidationService;
-    this.conditionManager = conditionManager;
-    this.actionManager = actionManager;
-    log.info("Screen Controller Created!");
-  }
-
-  /**
-   * Chooses which screen template and model data to render.
-   *
-   * @param flow        The current flow name, not null
-   * @param screen      The current screen name in the flow, not null
-   * @param uuid        The uuid of a subflow entry, can be null
-   * @param httpSession The current httpSession, not null
-   * @return the screen template with model data
-   */
-  @GetMapping("{flow}/{screen}")
-  ModelAndView getScreen(
-      @PathVariable String flow,
-      @PathVariable String screen,
-      @RequestParam(required = false) Map<String, String> query_params,
-      @RequestParam(value = "uuid", required = false) String uuid,
-      HttpSession httpSession
-  ) {
-    log.info(String.format("/flow/%s/%s 🚀", flow, screen));
-    log.info("getScreen: flow: " + flow + ", screen: " + screen);
-
-    var currentScreen = getScreenConfig(flow, screen);
-    Submission submission = submissionRepositoryService.findOrCreate(httpSession);
-    if ((submission.getUrlParams() != null) && (!submission.getUrlParams().isEmpty())) {
-      submission.mergeUrlParamsWithData(query_params);
-    } else {
-      submission.setUrlParams(query_params);
+    private final List<FlowConfiguration> flowConfigurations;
```

Comment:
This is changing to 4 spaces, are you still using the google stylesheet?  Seems like Intellij might be dropping use of that in some circumstances.

Reply:
It's weird that it's just this file... I wonder what is going on here.

Reply:
After re-opening IntelliJ this morning, it asked me to revert the file to "2 spaces" and now the diff is much better 💆🏻‍♀️ ☮️